src/status_im/contexts/communities/events.cljs

```diff
@@ -69,7 +69,7 @@
      (let [{:keys [token-permissions
                    token-permissions-check joined id]
             :as   community}      (<-rpc community-js)
-           has-token-permissions? (not (seq token-permissions))]
+           has-token-permissions? (seq token-permissions)]
```

Comment:
I'm not sure if this is a problem here, but twice I faced a bug about treating the existence of `token-permissions` as a proxy for meaning *joining a token-gated community*. Instead, if we mean that, we should filter the permissions that match the membership type (member permission) and ignore others (admin permissions and channel permissions).